src/ontology/mondo-edit.obo

```diff
@@ -390377,12 +390377,14 @@ is_a: MONDO:0000467 {source="NCIT:C62017"} ! second-degree atrioventricular bloc
 
 [Term]
 id: MONDO:0020745
-name: autosomal dominant cardiac arrhythmia (Kuhn)
```

Comment:
We don't want to delete old labels, can you make it an exact synonym and mark it as A synonym that is historic and discouraged (has synonym type -> synonym type property)

thanks!